programmers/난이도별/level01.크레인_인형뽑기_게임/dkdlelk99.py

```diff
@@ -8,19 +8,27 @@ def solution(board, moves):
             if board[j][i] != 0:
                 row.append(board[j][i])
         new_board.append(row)
-
+    
     for i in moves:
         if new_board[i-1] == []:
             continue
         else:
-            element = new_board[i-1].pop(0)
             if basket == []:
-                basket.append(element)
-            elif basket[-1] != element:
-                basket.append(element)
-            elif basket[-1] == element:
-                answer += 1
-                basket.remove(basket[-1])
-    return answer*2
+                basket.append(new_board[i-1].pop(0))
+            else:
+                element = new_board[i-1].pop(0)
+                if basket[-1] != element:
+                    basket.append(element)
+                elif basket[-1] == element:
+                    answer += 2
+                    basket.pop()
```

Comment:
```suggestion
                basket.append(new_board[i-1].pop(0))
                continue
            element = new_board[i-1].pop(0)
            if basket[-1] != element:
                basket.append(element)
            elif basket[-1] == element:
                answer += 2
                basket.pop()
```

if의  depth는 최대한 얕게 유지해주세요!

Reply:
if문 안에 여러 단계로 if문이 많지 않아야 한다는 말씀이죠?
리뷰 감사합니다!